build.gradle

```diff
@@ -121,12 +121,12 @@ subprojects {
     }
   }
   // aggregated task for checkstyle and spotbugs static analyzers
-  task('analyze') {
+  /*task('analyze') {
```

Comment:
Commenting out this task causes [a CI failure](https://app.circleci.com/pipelines/github/linkedin/cruise-control/1360/workflows/0b62aaf0-6417-4899-bca1-5836949b574c/jobs/4570) -- can we uncomment it and address the corresponding style issues brought up by this task (if any)?
```
* What went wrong:
Task 'analyze' not found in root project 'cruise-control'.
```